front/js/service.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * Create a scratchy service.
+ * @constructor
+ * @param {string} apiUrl - Scratchy service API URL, eg: "http://localhost:5000"
+ * @param {string} roomID - room id , eg:60895dd62d1a706830c31f10
```

Comment:
They are no `roomID` argument for `function ScratchyService(apiUrl)`, you need to remove it from the jsdoc.